src/Symfony/Bridge/Doctrine/Security/RememberMe/DoctrineTokenProvider.php

```diff
@@ -63,7 +63,7 @@ public function loadTokenBySeries(string $series): PersistentTokenInterface
 
         [$class, $username, $value, $last_used] = $row;
 
-        return new PersistentToken($class, $username, $series, $value, new \DateTimeImmutable($last_used));
+        return new PersistentToken($class, $username, $series, $value, new \DateTimeImmutable($last_used), false);
```

Comment:
will this code be compatible with 8.0? it'd be great
should we instantiate using the new signature a feature test?